test/lib/linter/xmlTemplate/lib/JSTokenizer.ts

```diff
@@ -14,6 +14,15 @@ test("JSTokenizer: Basic values", (t) => {
 	});
 });
 
+test("JSTokenizer: Tab after key", (t) => {
+	const res = JSTokenizer.parseJS(
+		`{path	: 'invoice>Status', formatter: 'ui5.walkthrough.model.formatter.statusText'}`);
```

Comment:
would you mind using \t for clarity?

Reply:
Done. Good point. I also added another test case with a quoted key, which did already work before. Only the unquoted failed.